admin.php

```diff
@@ -41,218 +41,191 @@ function sqlQuery($statement, $link)
+                                $row = sqlQuery("SELECT gl_value FROM globals WHERE gl_name = 'openemr_name' LIMIT 1", $dbh);
+                                $openemr_name = $row ? $row['gl_value'] : '';
+
+                                // Get version indicators from the database.
+                                $row = sqlQuery("SHOW TABLES LIKE 'version'", $dbh);
+                                if (empty($row)) {
+                                    $openemr_version = 'Unknown';
+                                    $database_version = 0;
+                                } else {
+                                    $row = sqlQuery("SELECT * FROM version LIMIT 1", $dbh);
+                                    $database_patch_txt = "";
+                                    if (!(empty($row['v_realpatch'])) && $row['v_realpatch'] != 0) {
+                                        $database_patch_txt = " (" . $row['v_realpatch'] . ")";
+                                    }
+
+                                    $openemr_version = $row['v_major'] . "." . $row['v_minor'] . "." .
+                                    $row['v_patch'] . $row['v_tag'] . $database_patch_txt;
+                                    $database_version = 0 + $row['v_database'];
+                                    $database_acl = 0 + $row['v_acl'];
+                                    $database_patch = 0 + $row['v_realpatch'];
+                                }
+
+                                // Display relevant columns.
+                                echo "  <td>$openemr_name</td>\n";
+                                echo "  <td>$openemr_version</td>\n";
+                                if ($v_database != $database_version) {
+                                    echo "  <td><a href='sql_upgrade.php?site=$sfname' class='text-decoration-none'>Upgrade Database</a></td>\n";
+                                } elseif (($v_acl > $database_acl)) {
+                                    echo "  <td><a href='acl_upgrade.php?site=$sfname' class='text-decoration-none'>Upgrade Access Controls</a></td>\n";
+                                } elseif (($v_realpatch != $database_patch)) {
+                                    echo "  <td><a href='sql_patch.php?site=$sfname' class='text-decoration-none'>Patch Database</a></td>\n";
+                                } else {
+                                    echo "  <td><i class='fa fa-check fa-lg text-success' aria-hidden='true' ></i></a></td>\n";
+                                }
+                                if (($v_database == $database_version) && ($v_acl <= $database_acl) && ($v_realpatch == $database_patch)) {
+                                    echo "  <td><a href='interface/login/login.php?site=$sfname' class='text-decoration-none'><i class='fa fa-sign-in fa-lg' aria-hidden='true' data-toggle='tooltip' data-placement='top' title ='Login to site $sfname'></i></a></td>\n";
```

Comment:
@stu01509 , There should be a login link here. Note this is not your fault, but is because the fontawesome icon needs to change. need to use `fa-sign-in-alt` instead of `fa-sign-in` above
![image](https://user-images.githubusercontent.com/278968/83960174-8750a400-a83a-11ea-9f72-7a37ce05e46f.png)

Reply:
Thanks for your mention, I fixed it.
It looks like good.

![image](https://user-images.githubusercontent.com/22230889/83960490-e60a3200-a8bb-11ea-9e01-f87890eb09a4.png)